egs/librispeech/ASR/lstm_transducer_stateless2/decode.py

```diff
@@ -768,6 +790,15 @@ def main():
     model.to(device)
     model.eval()
 
+    lm_filename = "5ram.fst.txt"
```

Comment:
Could you change it so that it is passed from the command line?

Reply:
Done